FFXIVClientStructs/FFXIV/Component/GUI/AtkImageNode.cs

```diff
@@ -36,6 +36,7 @@ public void Ctor() {
 public enum ImageNodeFlags {
     FlipH = 0x01,
     FlipV = 0x02,
+    RenderMiniMap = 0x04, // Renders minimap using the set textures as a mask
```

Comment:
This flag has nothing to do with the minimap and is a different type of flag that just happens to be set on the minimap.